README.md

```diff
@@ -12,29 +12,52 @@
 
 <!--te-->
 
```

Comment:
As a future update is nice to add CI and pictograms that reflect the status of the build.
![Selection_152](https://user-images.githubusercontent.com/11608884/94585801-2b96a480-0289-11eb-9274-5e0a99a6cce8.png)